buildSrc/src/main/java/org/opensearch/gradle/pluginzip/Publish.java

```diff
@@ -37,27 +41,26 @@ public static void configMaven(Project project) {
                 });
             });
             publishing.publications(publications -> {
-                final Publication publication = publications.findByName(PUBLICATION_NAME);
-                if (publication == null) {
-                    publications.create(PUBLICATION_NAME, MavenPublication.class, mavenZip -> {
-                        String zipGroup = "org.opensearch.plugin";
-                        String zipArtifact = project.getName();
-                        String zipVersion = getProperty("version", project);
-                        mavenZip.artifact(project.getTasks().named("bundlePlugin"));
-                        mavenZip.setGroupId(zipGroup);
-                        mavenZip.setArtifactId(zipArtifact);
-                        mavenZip.setVersion(zipVersion);
-                    });
-                } else {
-                    final MavenPublication mavenZip = (MavenPublication) publication;
-                    String zipGroup = "org.opensearch.plugin";
-                    String zipArtifact = project.getName();
-                    String zipVersion = getProperty("version", project);
-                    mavenZip.artifact(project.getTasks().named("bundlePlugin"));
-                    mavenZip.setGroupId(zipGroup);
-                    mavenZip.setArtifactId(zipArtifact);
-                    mavenZip.setVersion(zipVersion);
+                MavenPublication mavenZip = (MavenPublication) publications.findByName(PUBLICATION_NAME);
+
+                if (mavenZip == null) {
+                    LOGGER.info("{} publication not found, creating a new one", PUBLICATION_NAME);
```

Comment:
Do we really need to log that new publication is created? I doubt it has any value for majority of the pluging developers ...

Reply:
@reta I was thinking about that too. I was a bit struggling to configure logging correctly for gradle plugin tests anyway (meaning I was not able to get the LOGGER working correctly and I ended up using good-old-school `System.out.println();` ...  yea, I know 🤦 ).

I am fine to remove this line.